### PR TITLE
Refs[TASK][37991]Tạo giao diện hiển thị danh sách tướng yêu thích

### PR DESCRIPTION
**Pull request by mithong**
## Related Tickets
- [#Task 37991: Content](https://edu-redmine.sun-asterisk.vn/issues/37991)
## WHAT
- Hiển thị danh sách các hero yêu thích của người dùng
## Evidence (Screenshot or Video)
image
## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/item_layout_favorite.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                app:layout_constraintEnd_toEndOf="parent"
+                app:strokeColor="@color/color_C29E52"
+                app:strokeWidth="@dimen/dp_2">
+
+                <ImageView
+                    android:id="@+id/imageViewLikeFavorite"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:background="@android:color/black"
+                    android:src="@drawable/ic_baseline_favorite_24" />
```
Comment (khoaqh-2470) [inline app/src/main/res/layout/item_layout_favorite.xml]:
```suggestion
                    android:src="@drawable/ic_favorite" />
```
Đổi lại tên file icon nhé

**Review by khoaqh-2470 — APPROVED** [review]

File: app/src/main/res/layout/item_layout_favorite.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:background="@color/color_081B27"
+            android:layout_width="@dimen/_80sdp"
+            android:layout_height="wrap_content">
+
+            <com.google.android.material.card.MaterialCardView
+                android:id="@+id/materialCardView"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                app:layout_constraintWidth_percent="1"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_layout_favorite.xml]: percent 1 thì match_parent

Comment (tuanlvt-1117) [reply app/src/main/res/layout/item_layout_favorite.xml]: remove percent

File: app/src/main/res/layout/item_layout_favorite.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:background="@color/color_081B27"
+            android:layout_width="@dimen/_80sdp"
+            android:layout_height="wrap_content">
+
+            <com.google.android.material.card.MaterialCardView
+                android:id="@+id/materialCardView"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                app:layout_constraintDimensionRatio="1:1"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_layout_favorite.xml]: code dư

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**